seance/discord_bot/__init__.py

```diff
@@ -744,6 +741,61 @@ async def on_presence_update(self, _before: Member, after: Member):
         self._cached_status = status
 
 
+    # Needs to be raw because message might not be in the message cache.
```

Comment:
Can you not use `self._refetch_message()`?

Reply:
This is an event handler, it simply won't be called at all (we tried it), personally it seems like a bit of an odd choice but that's a discord.py choice